main.py

```diff
@@ -33,6 +37,32 @@ def cli_entrypoint(args):
     exit(1)
 
 
+def __build_configuration(config_file: str):
+  file_config = {}
+  if os.path.exists(config_file):
+    with open(config_file, "r", encoding="utf-8") as f:
+      file_config = json.loads(f.read())
+    if file_config.get("red_key") == "xxxxxxxxxx":
+      file_config.pop("red_key")
+    if file_config.get("ops_key") == "xxxxxxxxxx":
+      file_config.pop("ops_key")
+
+  env_vars = {
+    key: value
+    for key, value in {
+      "inject_torrents": Config.boolify_string_option(os.getenv("INJECT_TORRENTS")),
+      "injection_link_directory": os.getenv("INJECTION_LINK_TORRENTS"),
+      "deluge_rpc_url": os.getenv("DELUGE_RPC_URL"),
+      "qbittorrent_url": os.getenv("QBITTORRENT_URL"),
+      "red_key": os.getenv("RED_KEY"),
+      "ops_key": os.getenv("OPS_KEY"),
+    }.items()
+    if value
+  }
+
+  return Config([env_vars, file_config])
```

Comment:
priority is just controlled by order here, higher index in list means higher preference for option